test/scenarios/rhel-9.2-microshift-source-rollback.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Sourced from scenario.sh and uses functions defined there.
+
+scenario_create_vms() {
+    prepare_kickstart host1 kickstart.ks.template rhel-9.2-microshift-source
+    launch_vm host1
+}
+
+scenario_remove_vms() {
+    remove_vm host1
+}
+
+scenario_run_tests() {
+    run_tests host1 \
+        --variable "FAKE_NEXT_MINOR_REF:rhel-9.2-microshift-source-fake-next-minor" \
```

Comment:
No, but it's up to "operator" to set up "next fake minor" instead of another alias to "microshift-source" (which would work but current approach explicitly checks if it's in fact restored, "microshift-source" could be migrated)